CHANGES.rst

```diff
@@ -9,6 +9,8 @@ Changes by Version
   ``tchannel.sync.thrift.client_for`` as planned in 0.18.
 - **BREAKING** - Removed ``tchannel.thrift_request_builder`` as
   planned in 0.18.
+- **BREAKING** - ``tchannel.thrift.register`` returns the wrapped function
```

Comment:
Suggest renaming to "Enable unit testing endpoints by calling them as functions directly. This is enabled by ``tchannel.thrift.register`` returning..." or something like it.